### DISCUSSION TITLE
Stopped receiving emails from Github Explore.

### DISCUSSION THREAD

**Discussion by lucassmcardoso** [community/community#67002] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Since september 6 I've stopped receiving emails from the Github Explore with the "Here's what we found  based on your interests...". Does anyone know if there've been some change to the subscription model? If not, can anyone tell me where can I find the subscription page on the Github site?

As of now, I'm having to rely on entering the explore page on Github. I'd be very grateful for any answers on this.

**Comment by ILiuhao** [comment]
me to.

**Reply by celikmustafa89** [reply]
me too, same day the 6 th of September. 
If you have the same problem in same day, then they probably remove that feature. It is very sad :((

**Comment by makumi10** [comment]
Check Your GitHub Notification Settings:

Go to your GitHub account settings by clicking on your profile picture in the top right corner and selecting "Settings."
In the left sidebar, click on "Notifications."
Ensure that your email notification preferences are correctly configured. Specifically, make sure that "Participating" and "@mentions" are set to "Email" and "On GitHub.
Sometimes, GitHub notification emails can end up in your spam or junk folder. Check these folders to see if the emails are being filtered there.

**Reply by celikmustafa89** [reply]
Everything is up to date. I think they have changed something in GitHub. because my last email notification is on September 6 like the others. this can not be a coincidence or something we accidentally removed or changed.

**Comment by HowieWon** [comment]
Same with the Sep 5th

**Comment by MC-dusk** [comment]
Me too, but since February 28th, 2023

**Comment by h4rk8s** [comment]
me too
